service.go

```diff
@@ -218,6 +222,21 @@ func (self *Service) GetConfig() (*ServiceConfig, error) {
 	}, nil
 }
 
+func (self *Service) GetDelayedAutoStartInfo() (*ServiceDelayedAutoStartInfo, error) {
+	var res wrappers.SERVICE_DELAYED_AUTO_START_INFO
+	size := uint32(unsafe.Sizeof(res))
+	err := wrappers.QueryServiceConfig2(
+		self.handle,
+		wrappers.SERVICE_CONFIG_DELAYED_AUTO_START_INFO,
+		(*byte)(unsafe.Pointer(&res)),
+		size,
+		&size)
+	if err != nil {
+		return nil, NewWindowsError("QueryServiceConfig2", err)
+	}
+	return &ServiceDelayedAutoStartInfo{DelayedAutoStart: res.DelayedAutostart == 1}, nil
```

Comment:
[Minor] Let's make this `res.DelayedAutostart != 0`.  The Win32 `BOOL` data type treated anything other than 0 as true.